jekyll/_cci2/bundling-config.md

```diff
@@ -0,0 +1,905 @@
+
+### Command Syntax
+{:.no_toc}
+
+A command can have the following immediate children keys as a map:
+
+- **description:** (optional) A string that describes the purpose of the command, used for generating documentation.
+- **parameters:** (optional) A map of parameter keys, each of which adheres to the `parameter` spec.
+- **steps:** (required) A sequence of steps run inside the calling job of the command.
+
+### Defining Your Commands
+{:.no_toc}
+
+Commands are declared under the `commands` key of a `config.yml` or `orb.yml` file. For instance, to make a command called `sayhello` it might look like:
+
+```yaml
+commands:
+  sayhello:
+    description: "A very simple command for demonstration purposes"
+    parameters:
+      to:
+        type: string
+        default: "Hello World"
+    steps:
+      - run: echo << parameters.to >>
+```
+
+When invoking a command, the steps of that command are inserted where it's invoked. Commands can only be invoked as part of the sequence under `steps` in a job.
+
+### Invoking Other Commands in Your Command
+{:.no_toc}
+
+Commands can use other commands in the scope of execution. 
+
+<!--
+For instance, if a command is declared inside your Orb it can use other commands in that orb. It can also use commands defined in other orbs that you have imported (for example `some-orb/some-command`).
+-->
+
+## Built-in Commands
+
+CircleCI has several built-in commands available to all [circleci.com](http://circleci.com) customers and available by default in CircleCI server installations. Examples of built-in commands are:
+
+  * `checkout`
+  * `setup_remote_docker`
+  * `save_to_workspace`
+
+**Note:** Built-in commands are implicitly in the empty scope and are thus syntactically equivalent to primitives such as `run`). This _may_ change in future versions of configuration but is true to maintain compatibility with version `2` configuration.
+
+## Examples
+
+<!---
+The following is a an example of part of an "s3tools" orb defining a command called "s3sync":
+
+```yaml
+# s3tools orb
+commands:
+  s3sync:
+    description: "A simple encapsulation of doing an s3 sync"
+    parameters:
+      from:
+        type: string
+      to:
+        type: string
+      overwrite:
+        default: false
+        type: boolean
+    steps:
+      - run:
+          name: Deploy to S3
+          command: aws s3 sync << parameters.from >> << parameters.to >><<# parameters.overwrite >> --delete<</ parameters.overwrite >>"
+```
+--->
+
+Defining a command called "s3sync" is invoked in a 2.1 `.circleci/config.yml` file as:
+
+```yaml
+version: 2.1
+
+orbs:
+  s3tools: circleci/s3@1
+
+workflows:
+  build-test-deploy:
+    jobs:
+      - deploy2s3:
+        steps:
+          - s3tools/s3sync:
+              from: .
+              to: "s3://mybucket_uri"
+              overwrite: true
+```
+
+If the same command were declared locally inside `config.yml` it would look something like:
```

Comment:
I think we could start after this line and then delete the example above -- that way it's just about commands local to config.yml

Reply:
thanks, fixed.